test/testdata/resolver/overrides.rb

```diff
@@ -56,6 +56,14 @@ class B5 < A5
   def foo; end
 end
 
+# however, not using override when overriding B5#foo should be an error
+class C5 < B5
+  extend T::Sig
+  sig {void}
+  def foo; end
+# ^^^^^^^ error: Method `C5#foo` overrides an overridable method
+end
```

Comment:
This test case builds on previous classes present in this file: the parent class B5 has a valid method defined with `override`.